packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -4,13 +4,16 @@ set -e
 
 systemctl_stop() {
     unit="$1"
-    for i in $(seq 10); do
+
+    echo "Stoping $unit [attempt $i]"
```

Comment:
`$i` is no longer a variable here, perhaps you meant just `echo "Stopping $unit"`